mautic/mautic
# Fix email translations lost and overwritten by parent

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

This PR is follow up to https://github.com/mautic/mautic/pull/10525  and fix https://github.com/mautic/mautic/issues/10618
It's fix email translations lost and overwritten by parent 
Require https://github.com/mautic/mautic/pull/10619 



#### Steps to test this PR:


1. Load up [this PR](https://mautibox.com)
2. Create  segment email  with French language (`Subject FR + content Test FR`)
3. Clone it and set as translation of email `Subject FR` with language EN (`Subject EN + content Test FR`)
4.  Save and  go to email `Subject FR` and re-save.
5. This should overwrite content of email `Subject EN`. Your content of EN email should be Test FR
6. After PR should be OK




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/10620"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
